src/content/reference/react/index.md

```diff
@@ -56,11 +53,11 @@ function Form() {
 
 ---
 
-## Effect Hooks {/*effect-hooks*/}
+## Hook Efek {/*effect-hooks*/}
```

Comment:
Effect di sini refer ke API nya React.
Gak usah diterjemahkan apa ya? 
CC: @mhaidarhanif @resir014 @r17x @zainfathoni

Reply:
"Effect" dalam tulisan ini konteksnya grouping dan juga istilah. 

Menurutku masih okay untuk ditranslate kecuali API Efek seperti `useEffect`

Reply:
Need to confirm which one should i use

Reply:
Then it's okay to translate

Reply:
thx for the confirmation, i'll make this conversation resolve then